### ISSUE TITLE
Add screen reader labels to icon buttons

### ISSUE THREAD

**Issue by fishdev** [QuantumStack/Descartes-client#15]
Pages such as the dashboard student course view use buttons that do not have associated text. They should have an [`aria-label`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) attribute to improve accessibility.
